addons/website_mass_mailing/static/tests/tours/newsletter_popup.js

```diff
@@ -44,13 +44,12 @@ tour.register('newsletter_popup_use', {
 }, [
     {
         content: "Check the modal is not yet opened and force it opened",
-        trigger: '.o_newsletter_popup',
+        trigger: 'body:has(.o_newsletter_popup)',
         run: function (actions) {
             const $modal = this.$anchor.find('.modal');
             if ($modal.is(':visible')) {
                 console.error('Modal is already opened...');
             }
-            $(document).trigger('mouseleave');
```

Comment:
How does it work without this? :thinking: Maybe the modal is in delay mode and this is is the remaining of a test which tested the "on exit" mode?